src/main/kotlin/de/amplimind/codingchallenge/controller/AdminController.kt

```diff
@@ -45,12 +39,22 @@ class AdminController(
     @ApiResponse(responseCode = "404", description = "User with email was not found.")
     @GetMapping("fetch/projects/{email}")
     fun fetchUserInfosForEmail(
-        @PathVariable email: String,
+            @PathVariable email: String,
     ): ResponseEntity<UserInfoDTO> {
         val userInfo = this.userService.fetchUserInfosForEmail(email)
         return ResponseEntity.ok(userInfo)
     }
 
+    @Operation(summary = "Endpoint for deleting a user by email")
+    @ApiResponse(responseCode = "200", description = "User was deleted successfully.")
+    @DeleteMapping("user/{email}")
+    fun deleteUserByEmail(
+            @PathVariable email: String,
+    ): ResponseEntity<UserInfoDTO> {
+        this.userService.deleteUserByEmail(email)
+        return ResponseEntity.ok().build()
```

Comment:
jz muss nur noch die returnte instanz vom UerInfoDTO (von this.userService.deleteUserByEmail(email) ) mit dem ResponseEntity mitgegeben werden ^^